Kitodo/src/main/webapp/pages/Indexing.xhtml

```diff
@@ -32,10 +44,40 @@
                             PF('progressPoll').stop();
                             $('.refreshTable').click();
                         }
+                        else if (message.includes("mapping")) {
```

Comment:
I would prefer a similar formatting style for JavaScript and Java. This would mean, that `else` blocks should begin in them line as normal `if` block ends.
I'm open for other opinions as I did not wrote many JavaScript based code.